src/containers/pages/InterventionPlan/IRS/index.tsx

```diff
@@ -127,9 +127,7 @@ class IrsPlans extends React.Component<IrsPlansProps & RouteComponentProps<Route
           );
           return fetchPlanRecordsActionCreator(irsPlanRecords);
         })
-        .catch(err => {
-          // console.log('ERR', err)
-        });
+        .catch(); // is this wise?
```

Comment:
> is this wise?

I'm guessing not (and neither was previous implementation) because it's not actually doing anything with the error (other than avoiding a `TypeError` crash). Is there any reason we don't `growl` an error message?

Reply:
I somehow missed it, will growl this.